EST.MIT.InvoiceImporter.Function/Functions/ImporterFunctions.cs

```diff
@@ -57,4 +71,21 @@ public ImporterFunctions(IAzureBlobService azureBlobService, IAzureTableService
             throw;
         }
     }
+
+    private Notification CreateNotificationRequest(ImportRequest importMessage)
+    {
+        return new NotificationBuilder()
+                                        .WithId(importMessage.ImportRequestId.ToString())
+                                        .WithScheme(importMessage.SchemeType)
+                                        .WithAction(importMessage.Status)
```

Comment:
I don't think importMessage.Status matches up with the NotificationType enum....?

Reply:
We discussed this. Is probably OK.